Trackify/Trackify/Flight.swift

```diff
@@ -10,14 +10,27 @@ import Foundation
 import AWSDynamoDB
 
 class Flight : AWSDynamoDBObjectModel, AWSDynamoDBModeling {
+    fileprivate var df = DateFormatter()
+    fileprivate var timeString: String?
+    fileprivate var dateString: String?
+    fileprivate var date: Date?
+    
     var airline: String?
     var flightNumber: String?
-    var date: Date?
     var departureAirport: String?
     var destinationAirport: String?
     var confirmation: String?
-    var email:String?
-    var datetime:String?
+    var email: String?
+    var datetime: String? {
+        didSet {
+            df.dateFormat = "MM-dd-yyyy HH:mm"
+            self.date = df.date(from: datetime!)
+            df.dateFormat = "h:mm a"
+            self.timeString = df.string(from: self.date!)
+            df.dateFormat = "MMMM d, yyyy"
+            self.dateString = df.string(from: self.date!)
```

Comment:
This is so that the time and date display nicely in the flights table. The "a" at the end of the time format adds "AM" or "PM" to the string and the "MMMM d, yyyy" date format makes the date display as "February 10, 2016" rather than "02-10-2016"